src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -22,69 +23,57 @@
       {% if reason === NotEligibleReason.CLAIM_VALUE_NOT_KNOWN %}
         <p>{{ t('You need to know the claim amount to use this service.') }}</p>
         <p>{{ t('If you can’t calculate the claim amount, for example because you’re claiming for an injury or accident, ' +
-        ' use the <a target="_blank" href="{{ n1FormUrl }}">N1 paper form</a> '+ newWindow() + '.' ,
-          {  n1FormUrl : n1FormUrl }) | safe }}</p>
+        ' use the ' + externalLink('N1 paper form', n1FormUrl) + newWindow() + '.') | safe }}</p>
```

Comment:
Please remove leading whitespace, there is already one at the end of line above.